### ISSUE TITLE
Seperating the title with the subcontent in about (Sidebar)

### ISSUE THREAD

**Issue by TenzDelek** [nodejs/nodejs.org#6677]
### URL:

https://nodejs.org/en/about

### Browser Name:

brave

### Browser Version:

1.65.123

### Operating System:

Windows 11

### How to reproduce the issue:

1. Go the about 
the current visual looks like this 
<img width="274" alt="image" src="https://github.com/nodejs/nodejs.org/assets/122612557/274e79e5-b51c-4ab1-b1dc-15bd785d2135">
<img width="235" alt="image" src="https://github.com/nodejs/nodejs.org/assets/122612557/cefb5e21-9de1-4c85-8d29-c42f08ae9beb">

i notice a visual dissatisfaction with the title and the subpart of it . i feel like there should be more separation between them  (not only making the font small we can try the below idea ) 
solution:
1. either darken out the title a little bit to show a contrast to it
2. or we can add a padding left to the subpart to show seperation

**Comment by TenzDelek** [comment]
solution : example for the first one :
<img width="315" alt="image" src="https://github.com/nodejs/nodejs.org/assets/122612557/4718d79d-c1c7-4e41-a49d-de683b10753a">
for the second one:
<img width="298" alt="image" src="https://github.com/nodejs/nodejs.org/assets/122612557/4799f86d-ab84-4c42-a56d-c0577af153cb">

**Comment by AugustinMauroy** [comment]
I allow it's not 100% clear but I thinks the best things it's just to add bottom border to the title.

**Comment by TenzDelek** [comment]
@AugustinMauroy  ya the border bottom one  sounds good, will create a pr for it

**Comment by ovflowd** [comment]
It's subject to personal interpretation. For me, it is already clear that one is a section title and the latter are section entries. I also believe that the suggestions aren't really good ones.

I'm not even sure this is an accessibility issue, @AugustinMauroy why did you mark it as a accessibility issue?

**Comment by AugustinMauroy** [comment]
> I'm not even sure this is an accessibility issue, @AugustinMauroy why did you mark it as a accessibility issue?

I made this choice because the issue isn't a bug (unexpected behaviour) but a problem of readability and the level of highlighting of elements
